CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+Each accepted governance action will be assigned a unique identifier (a.k.a. the **governance action ID**), consisting of the transaction ID that created it and the index within the transaction body that points to it.
+
+### Votes
+
+Each vote transaction consists of the following:
+
+* a governance action ID;
+* a role - Constitutional Committee, DRep, or SPO;
+* a key-hash (which will be verified to have the role above);
+* a URL for any metadata that is relevant to the vote;
+* a hash of the contents of this URL;
+* a yes/no/abstain vote.
+
+Note that "abstained" votes are not included in the "active voting stake".
+Note also that a vote to abstain is different than abstaining from voting
+(the former is visible on chain, the latter is not).
+To avoid confusion, we will only use the word "abstain" from this point onward to mean a vote
+to abstain.
+
+The key hash will trigger the appropriate signature check on the transaction body according to the existing `UTxOW` ledger rule.
+
+Votes can be cast multiple times for each governance action by a single key hash. Correctly submitted votes override any older votes for the same key hash and role.
+As soon as a governance action is ratified, voting ends. Any future votes are not considered or recorded (whether they are yes, no or abstain).
+
+#### Governance State
+
+When a governance action is successfully submitted to the chain, its progress will be tracked by the ledger state.
+In particular, the following will be tracked:
+
+* the governance action ID;
+* the epoch that the action expires;
+* the deposit amount;
+* the rewards address that will receive the deposit when it is returned;
+* the total yes/no/abstain votes of the Constitutional Committee for this action;
+* the total yes/no/abstain votes of the DReps for this action;
+* the total yes/no/abstain votes of the SPOs  for this action.
+
+#### Stale Votes
+
+The votes from the DReps and the SPOs may become meaningless after crossing an epoch boundary since they may become unregistered. Therefore, all unregistered votes are cancelled before any new votes are considered.
+
+#### Changes to the Stake Snapshot
+
+Since the stake snapshot changes at each epoch boundary, a new current voting tally must be calculated for each governance action based on the votes that have been cast before any new votes are counted.  This avoids the same stake being used two or more times.  _This new tally may result in immediate ratification of a governance action._
+
+### Delegated Representatives (DReps)
+
+> **Warning**
+> The design of the DReps is still subject to change and should not be conflated with Project Catalyst DReps.
+
+Existing stake keys will be given new responsibilities. They will be able to delegate their stake to DReps. DRep Registration will mimic the existing stake delegation mechanisms (via certificates).
+
+The following new types of voting certificates will be added:
+
+* DRep registration certificate. Including:
```

Comment:
While writing out some details I discovered that a DRep registry isn't actually required from the ledger's perspective. We could simply have anyone delegate an arbitrary hash without checking if that hash actually belongs to a DRep (note that stake pool delegation also doesn't check whether the hash belongs to an actual pool). Tallying the DRep vote doesn't change much, as we simply have to do one less check.

I've discussed this with @jaredcorduan already, and it seems that has a bunch of technical benefits including dropping the requirement of a deposit and simplifying the ledger, with the only downside being that there is no formal declaration of intent to be a DRep being part of the ledger rules.

However, we could try to establish a metadata standard instead. This has the advantage of giving the control over metadata to the community, but of course people don't actually have to follow such a standard. That's probably fine though: DReps who don't play by the rules aren't discoverable, so they'd only get to vote with their own stake.

Are there any other good arguments for either options of the design space?